modules/shared/src/main/scala/org/tessellation/ext/cats/effect/package.scala

```diff
@@ -0,0 +1,9 @@
+package org.tessellation.ext.cats
+
+import _root_.cats.effect.{IO, Resource}
```

Comment:
Where does this `_root_` package comes from? Are we repackaging cats?

Reply:
@tbekas  You can't use `cats.effect.{IO, Resource}` because you are already in `cats` package `org.tesselation.ext.cats` so it will try to find `IO` and `Resource` in `org.tesselation.ext.cats.{IO, Resource}` (relatively).

`_root_` is a syntax to force absolute import instead of relative one